### ISSUE TITLE
Problem in part of the translation

### ISSUE THREAD

**Issue by mauricioros** [smalot/pdfparser#273]
Hello!
First of all thanks for making this amazing tool available!
Let's get to the problem .... I'm creating a small application using PDFParser.
But it cannot convert part of a pdf.

Can you help me?

I'm reading the following url:
https://dje.tjsp.jus.br/cdje/consultaSimples.do?cdVolume=14&nuDiario=2986&cdCaderno=12&nuSeqpagina=2993

And in the first occurrence of "REQDO" it should bring forward "Guaraná e Ramos Empreendimentos Imobiliários Ltda.", However bring the following set of characters "<001d0003002a00580044005500440051006900030048000300000000000000000000000000000000000000000000000000000000000000003000000000000000000030000000
440011 ["

Using the demo guys (https://www.pdfparser.org/demo) it also presents the same error.

Can you help me? Please!!!

**Comment by Connum** [comment]
The hexadecimal string in this position contains a newline character that causes this issue. I'm working on a fix that strips any newlines from hexadecimal strings bevore trying to decode them.

**Comment by Connum** [comment]
I just created a PR with a fix! There's this other decoding issue when testing the provided file
```
EXEQTE  : Infinit���)�D�V�K�L�R�Q��&�R�P�p�U�F�L�R��H��'�L�V�W�U�L�E�X�L�G�R�U�D��(�L�U�H�O�i
ADVOGADO  : 67978/SP - Cleodilson Luiz Sforzin
```
which will be fixed as well once #342 is merged.

**Comment by k00ni** [comment]
#342 was merged.
